textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/TestConstants.java

```diff
@@ -65,4 +65,52 @@
         return Arrays.asList(tuple1, tuple2, tuple3, tuple4, tuple5);
                 
     }
+    
+    //sample created by team3

```

Comment:
@sandeepreddy602 , you can build the[ test-jar](https://maven.apache.org/plugins/maven-jar-plugin/examples/create-test-jar.html) for that purpose. Then everything should goes to `test` folder.